packages/backend/src/database/migrations/20220727122405_drop_issuer_fk.ts

```diff
@@ -0,0 +1,31 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+    if (await knex.schema.hasTable('openid_identities')) {
+        await knex.schema.alterTable('openid_identities', (tableBuilder) => {
+            tableBuilder.dropForeign('issuer');
+        });
+    }
+    await knex.schema.dropTableIfExists('openid_issuers');
+}
+
+export async function down(knex: Knex): Promise<void> {
+    if (!(await knex.schema.hasTable('openid_issuers'))) {
+        await knex.schema.createTable('openid_issuers', (tableBuilder) => {
+            tableBuilder.text('issuer').primary();
+        });
+        await knex('openid_issuers').insert({
+            issuer: 'https://accounts.google.com',
+        });
+    }
+    if (await knex.schema.hasTable('openid_identities')) {
+        await knex.schema.alterTable('openid_identities', (tableBuilder) => {
```

Comment:
We might need to delete all rows where the issuer isn't `https://accounts.google.com`